code/clue.js

```diff
@@ -1,59 +1,246 @@
+  weight: "1 kg",
+  color: "green",
+  image: "https://images.unsplash.com/photo-1549899615-ef2899a8c02d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=634&q=80",
+},
+{ name: 'Trophy',
+  weight: "2kg",
+  color: "plum",
+  image: "https://images.unsplash.com/photo-1579913741676-b000995a823f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=634&q=80",
+},
+{ name: 'Revolver',
+  weight: "0.5 kg",
+  color: "orange",
+  image: "https://images.pexels.com/photos/53351/firearm-handgun-revolver-gun-53351.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260",
+}]
 
-const rooms = []
+const rooms = [
+  {name: "The Dining Room",
+  location: "East Wing",
+  color: "coral",
+  image: "https://images.pexels.com/photos/2504911/pexels-photo-2504911.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260"
+  },
+  {name: "The Conservatory",
+  location: "West/South Wing",
+  color: "orchid",
+  image: "https://images.unsplash.com/photo-1524070960420-47602852d30f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=675&q=80"
+  },
+  {name: "The Kitchen",
+  location: "South/East Wing",
+  color: "mediumslateblue",
+  image: "https://images.pexels.com/photos/2972365/pexels-photo-2972365.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260"
+  },
+  {name: "The Study",
+  location: "North/West Wing",
+  color: "salmon",
+  image: "https://images.pexels.com/photos/1907784/pexels-photo-1907784.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260"
+  }, 
+  {name: "The Library",
+  location: "West Wing",
+  color: "sienna",
+  image: "https://images.unsplash.com/photo-1541123039768-be0b59340e7b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=634&q=80"
+  }, 
+  {name: "The Billiard Room",
+  location: "West/South Wing",
+  color: "tomato",
+  image: "https://images.unsplash.com/photo-1577579242319-bbd5b7159055?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2141&q=80"
+  }, 
+]
 
-// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
+//Variables
 const randomSelector = array => {
-  return array[Math.floor(Math.random() * array.length)]
+ return array[Math.floor(Math.random() * array.length)]
+}
+
+let killer;
+let room;
+let weapon;
+
+//Functions
+const showKiller = () => {
+
+  const killerImage = document.getElementById("killer-image");
+
+  killerImage.src = killer.image;
+  killerImage.classList.add("reveal-styling");
+
+  document.getElementById("killerCard").classList.add("flip-cards");
+  document.getElementById('killerCardBack').style.background = killer.color;
+  document.getElementById("killerAge").innerHTML = `Age: ${killer.age}`;
+  document.getElementById("killerOccupation").innerHTML = `Occupation: ${killer.occupation}`;
+  document.getElementById('killerName').innerHTML = `${killer.firstName} ${killer.lastName}`;
+
+  if (killer.suspect === "Mrs White") {
```

Comment:
Why just Mrs White?

Reply:
She has a white background and you cant see the textboxes very well since they are also white :)